api/src/org/apache/cloudstack/api/response/NicResponse.java

```diff
@@ -102,6 +102,14 @@
     @Param(description = "Id of the vm to which the nic belongs")
     private String vmId;
 
+    @SerializedName("nsxlogicalswitch")
+    @Param(description = "Id of the NSX Logical Switch (if NSX based), null otherwise", since="4.6.0")
+    private String nsxlogicalswitch;
```

Comment:
variable naming convention not being followed here. It should be `nsxLogicalSwitch`